server/database/db.js

```diff
@@ -1,5 +1,6 @@
 const { Pool } = require('pg');
 
 const pool = new Pool();
+pool.database = 'theworld';
```

Comment:
I don't think you need to add this if you have your .env set up
```suggestion
```

Reply:
Ah, left over from some troubleshooting